addons/payment/wizards/payment_link_wizard.py

```diff
@@ -42,6 +42,21 @@ def default_get(self, fields):
     description = fields.Char('Payment Ref')
     access_token = fields.Char(compute='_compute_values')
     company_id = fields.Many2one('res.company', compute='_compute_company')
+    available_acquirer_ids = fields.Many2many(
+        comodel_name='payment.acquirer',
+        string="Payment Acquirers Available",
+        compute='_compute_payment_acquirers',
```

Comment:
Should have said it before, but the guidelines for compute methods are `_compute + field name = _compute_available_acquirer_ids`

```suggestion
        compute='_compute_available_acquirer_ids',
```